frontend/packages/console-app/src/components/quick-starts/QuickStartPanelContent.tsx

```diff
@@ -27,18 +29,19 @@ const QuickStartPanelContent: React.FC<QuickStartPanelContentProps> = ({
   handleClose,
   activeQuickStartID,
 }) => {
-  const [scrollDirection, handleScrollCallback] = useScrollDirection();
   const { t } = useTranslation();
+  const [shadows, SetShadows] = React.useState<Shadows>(Shadows.none);
   const quickStart = quickStarts.find((qs) => qs.metadata.name === activeQuickStartID);
 
-  const headerClasses = classNames('co-quick-start-panel-content-head', {
-    'pf-u-box-shadow-sm-bottom':
-      scrollDirection && scrollDirection !== ScrollDirection.scrolledToTop,
+  const updateShadow: UpdateShadow = (shadow) => SetShadows(shadow);
```

Comment:
You can directly pass `setShadow` to the component. Why do you need to wrap it?

Reply:
updated